plugins/field-date/src/field_date.js

```diff
@@ -167,24 +175,50 @@ Blockly.FieldDate.prototype.updateEditor_ = function() {
 };
 
 /**
- * Create and show the date field's editor.
+ * Shows the inline free-text editor on top of the text along with the date
+ *    editor.
```

Comment:
nit: I don't think this needs to be indented.

Reply:
Fixed